docker-compose.yml

```diff
@@ -0,0 +1,44 @@
+version: "3"
```

Comment:
We already have `evm_loader/docker-compose-test.yml` file. Why we need this one?

Reply:
Removed